injectproxy/enforce.go

```diff
@@ -111,7 +111,9 @@ func (ms Enforcer) EnforceNode(node parser.Node) error {
 	return nil
 }
 
-func (ms Enforcer) enforceMatchers(targets []*labels.Matcher) []*labels.Matcher {
+// EnforceMatchers replaces similar matcher present in
```

Comment:
I would be specific what "similar" means, but it's good enough (:

Reply:
will keep that in mind.